src/nodes/display/BleachBypassNode.js

```diff
@@ -0,0 +1,28 @@
+import { addNodeElement, float, tslFn, vec3, vec4 } from '../shadernode/ShaderNode.js';
+import { min, max, mix } from '../math/MathNode.js';
+import { luminance } from './ColorAdjustmentNode.js';
+
+export const bleach = /*@__PURE__*/ tslFn( ( [ color, opacity = 1 ] ) => {
+
+	const base = color;
+	const lum = luminance( base.rgb );
```

Comment:
I think it's more correct to use `luminance()` here instead of using a custom computation.

Reply:
@donmccurdy It seems there is another place in the code where `luminance()` could be used. 

https://github.com/mrdoob/three.js/blob/93da17b807b8f0dd0cfabd0b45f8d4d8b2b6c7a4/examples/jsm/shaders/BleachBypassShader.js#L41-L43

The coefficients `0.25, 0.65, 0.1` seems inconsistent to me...

Reply:
Agreed! I suspect they're just loose approximations, maybe the original implementation didn't have a particular color space in mind. We can use the default luminance coefficients instead (as you've done).